platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/ConsensusImpl.java

```diff
@@ -1087,6 +1088,17 @@ private AddressBook getAddressBook() {
         return addressBook;
     }
 
+    /**
+     * Checks if the event has the same index and the one provided.
+     *
+     * @param e the event to check for euqality to index
+     * @param index the index to check for equality to the event's index
+     * @return true if the event has the same index as the one provided, false otherwise
+     */
+    private boolean hasIndex(@NonNull final EventImpl e, final int index) {
```

Comment:
Nit: I'm not a fan of single letter variable names. I'd personally change it to `event` (even though the rest of the file uses single letters). Up to you, though

Reply:
changed to `event`